modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -26,16 +26,16 @@ example
     └── source-crs <1>
         ├── PaoCatalogSource.yaml
         ├── PaoSubscription.yaml
-        ├── bell
+        ├── custom
         |   ├── apiserver-config.yaml
         |   └── disable-nic-lldp.yaml
         └── elasticsearch
             ├── ElasticsearchNS.yaml
             └── ElasticsearchOperatorGroup.yaml
 ----
-<1> The `sources-crs` subdirectory is in the same directory as the `kustomization.yaml` file.
+<1> The `source-crs` subdirectory is in the same directory as the `kustomization.yaml` file.
 
-. Update the required `PolicyGenTemplate` CRs to include references to the content you added in the `custom-crs` directory.
+. Update the required `PolicyGenTemplate` CRs to include references to the content you added in the `source-crs/custom` subdirectory.
```

Comment:
addresses the comment https://github.com/openshift/openshift-docs/pull/60970/files#r1334167378